package.json

```diff
@@ -42,8 +56,5 @@
         "./src/scss"
       ]
     }
-  },
-  "dependencies": {
-    "tslib": "^1.9.0"
```

Comment:
What happened to `tslib`?

Reply:
I discovered that it is installed as a dependency of:
injection-js
less
rxjs
tslint
tsutils
ng-packagr

and then gets deduped a bunch, so I figured it wasn't necessary to list it explicitly.